src/binary.cc

```diff
@@ -43,16 +43,8 @@ const char* GetSectionName(BinarySection sec) {
 }
 
 const char* NameSubsectionName[] = {
-  "module",
-  "function",
-  "local",
-  "label",
-  "type",
-  "table",
-  "memory",
-  "global",
-  "elemseg",
-  "dataseg",
```

Comment:
Can we keep this as is?    I think its usefull to be able to see the index of a given string based on its line offset (since I think the indexes here are relevant numbers in the binary format).

Reply:
Wrapped in `// clang-format off` and `on`.